src/test/java/uk/nhs/prm/deduction/e2e/models/ContinueRequestMessage.java

```diff
@@ -0,0 +1,43 @@
+package uk.nhs.prm.deduction.e2e.models;
+
+import uk.nhs.prm.deduction.e2e.utility.Resources;
+
+import java.util.UUID;
+
+import static uk.nhs.prm.deduction.e2e.utility.TestUtils.getUUIDAsUpperCaseString;
+
+public class ContinueRequestMessage {
+    private UUID conversationId;
+    private UUID messageId;
+    private String sourceGpOds;
+    private String destinationGpOds;
+    private String sourceGpAsid;
+    private String destinationGpAsid;
```

Comment:
These can be final :D 

```suggestion
    private UUID conversationId;
    private final UUID messageId;
    private final String sourceGpOds;
    private final String destinationGpOds;
    private final String sourceGpAsid;
    private final String destinationGpAsid;
```